deployment/ansible/group_vars/all

```diff
@@ -42,11 +42,10 @@ docker_version: "1.9.*"
 docker_py_version: "1.2.3"
 docker_options: "--storage-driver=aufs"
 
-sjs_host: "localhost"
-sjs_port: 8090
-sjs_container_image: "quay.io/azavea/spark-jobserver:0.6.1"
+akka_host: "localhost"
```

Comment:
Since we already had `geop_version` defined here I changed these to `geop_host` & `geop_port` to match.